addon/-private/system/store.js

```diff
@@ -605,6 +605,51 @@ Store = Service.extend({
 
     See [peekRecord](#method_peekRecord) to get the cached version of a record.
 
+    ### Retrieving Related Model Records
+
+    If you use an adapter such as Ember's default
+    [`JSONAPIAdapter`](http://emberjs.com/api/data/classes/DS.JSONAPIAdapter.html)
+    that supports the [JSON API specification](http://jsonapi.org/) and if your server
+    endpoint supports the use of an
+    ['include' query parameter](http://jsonapi.org/format/#fetching-includes),
+    you can use `findRecord()` to automatically retrieve additional records related to
+    the one you request by supplying an `include` parameter in the `options` object.
+
+    For example, given a `post` model that has a `hasMany` relationship with a `comment`
+    model, when we retrieve a specific post we can have the server also return that post's
+    comments in the same request:
+
+    ```
+    // app/routes/post.js
```

Comment:
Instead of writing the filename in a comment you can write it after the 3 back ticks and it will get rendered as a header for the code block.